### PR TITLE
[WIP] Update docs.md

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

File: _pages/docs.md [hunk 1]
```diff
@@ -51,9 +51,37 @@ repository](http://github.com/proxyapps/proxyapps.github.io) of the proxy apps
...
+```json
+[
+  { 
+    ...
+    "longdesc" : ["line1",
+                  "line2",
+                  "line3"]
+  }
+]     
+```
```
Comment (junghans) [inline _pages/docs.md:67]: @bhatele: Does our website support that?

**Comment by richards12** [comment]
I just had a thought. Could we implement a forms interface so that contributors wouldn’t need to deal with either git or json?  This might be a job for a proper web developer

**Comment by junghans** [comment]
Depends how my submission we expect...https://goo.gl/forms/d621RPET55TszmLF3

File: _pages/docs.md (hunk 1 above)
Comment (bhatele) [reply _pages/docs.md:67]: not sure

**Review by bhatele — APPROVED** [review]

**Pull request merged**